project/plugins.sbt

```diff
@@ -12,3 +12,6 @@ addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.6.1")
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.2")
 
 addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "0.9.7")
+
+// dependency tracker plugin - needed for snyk cli integration
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.8.2")
```

Comment:
just a minor thing, but you could use the latest version - `0.9.0`

Reply:
@desbo  Good spot! This should be remedied now.